### PR TITLE
User page

### PR DESCRIPTION
**Pull request by emmanuelle-c**

### CODE REVIEW

**Review by Coline1186 — APPROVED** [review]

File: client/src/assets/context/status.jsx
Comment (Alexandre78R) [inline client/src/assets/context/status.jsx:1]: Move file context in src and rename context status.jsx => auth.jsx

File: client/src/components/Header/Header.css
Comment (Alexandre78R) [inline client/src/components/Header/Header.css:1]: Rename connection in English please

Comment (Alexandre78R) [inline client/src/components/Header/Header.css:1]: Rename Home and banner in English please

Comment (Alexandre78R) [inline client/src/components/Header/Header.css:1]: Delete comments please

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

File: client/src/assets/context/status.jsx
Comment (emmanuelle-c) [reply client/src/assets/context/status.jsx:1]: done !

File: client/src/components/Header/Header.css
Comment (emmanuelle-c) [reply client/src/components/Header/Header.css:1]: changes done in jsx and css files

Comment (emmanuelle-c) [reply client/src/components/Header/Header.css:1]: comment is deleted ! :) thanks !

Comment (emmanuelle-c) [reply client/src/components/Header/Header.css:1]: done !

**Review by Alexandre78R — APPROVED** [review]

**Pull request merged**